README.md

```diff
@@ -164,8 +164,8 @@ To create a new virtual environment:
 
 If you plan to develop on Maestro, install the repository directly using:
 
-    pip install -r requirements.txt
-    pip install -e .
+    pip install poetry
```

Comment:
This might be a silly question -- should this be done in a virtual environment?

Reply:
Might also be worth linking to the poetry website?

Reply:
From what I saw, this only installed dependencies, it didn't install Maestro itself. Would we still need the `pip install -e .` command?

Reply:
> From what I saw, this only installed dependencies, it didn't install Maestro itself. Would we still need the `pip install -e .` command?

Ok -- I think I verified that this installs.



> This might be a silly question -- should this be done in a virtual environment?

It doesn't look like this should be in a virtualenv at least from my toying with poetry.

Reply:
So poetry should be installed system wide. It then creates and manages the virtualenv for you, kind of like pipenv. A `poetry install` will install the project, just like doing a `pip -e .`

Reply:
Yeah, I just figured that out. I'm basically going to configure it to make the environments in my `WORKON_HOME` directory. It might be worth noting that so folks can use virtualenvs for other projects?

Reply:
Sorry about mucking up the comments also, I was trying to get oriented with the new-ness haha

Might be worth also making a discussion thread about `poetry` best practices that we could port to docs.

Reply:
That's what the comments are for.

A discussion thread might indeed be a good idea.

Reply:
I'll start one once this gets merged -- I want to make an introduce yourself post as well. I think it might help with community building!